shell/platform/android/external_view_embedder/external_view_embedder.cc

```diff
@@ -137,7 +137,14 @@ void AndroidExternalViewEmbedder::SubmitFrame(
         // slighly larger. For example, {0.3, 0.5, 3.1, 4.7} becomes {0, 0, 4,
         // 5}.
         intersection_rect.set(intersection_rect.roundOut());
-        overlay_layers.at(view_id).push_back(intersection_rect);
+        if (overlay_layers.at(view_id).empty()) {
+          overlay_layers.at(view_id).push_back(intersection_rect);
+        } else if (intersection_rect.contains(
+                       overlay_layers.at(view_id).back())) {
```

Comment:
I was expecting the overlay layers to be associated with two different view_ids since there are two platform views, and the fab button intersects both.

Do you mind verifying why a single `view_id` contain the two overlay surfaces?